fission-web-server/library/Fission/Web/Server.hs

```diff
@@ -138,20 +138,32 @@ server ::
   -> Fission.Routes (AsServerT m)
 server appHost =
   Fission.Routes
-    { v2
+    { v3
+    , v2
     , heroku      = genericServerT Heroku.handler
     , latestDocs  = v2Docs
     , unversioned = serverV_
     , root        = pure NoContent
     }
 
   where
+    v3 =
+      genericServerT Fission.RoutesV3
+        { api  = serverV3
+        , docs = v3Docs
+        }
+
     v2 =
       genericServerT Fission.RoutesV2
         { api  = serverV2
         , docs = v2Docs
         }
 
+    serverV3 =
+      genericServerT Fission.V3
+        { user   = genericServerT User.handlerV3
```

Comment:
What are your thoughts on versioning the other endpoints like `ipfs`, `app`, and `auth`? Doesn't seem right on this PR, but maybe we will want to version/alias them to have a consistent set of routes?

Reply:
I don't know, maybe alias them to v1 or v0 sounds like a good idea.